examples/thermomechanics/thermal_deformation.cpp

```diff
@@ -61,15 +61,16 @@ void thermalDeformationExample( const std::string filename )
     //                Force model type
     // ====================================================
     using model_type = CabanaPD::PMB;
-    using thermal_type = CabanaPD::TemperatureDependent;
+    using thermal_type = CabanaPD::DynamicTemperature;
 
     // ====================================================
     //                 Particle generation
     // ====================================================
     // Does not set displacements, velocities, etc.
-    auto particles = std::make_shared<
-        CabanaPD::Particles<memory_space, model_type, thermal_type>>(
-        exec_space(), low_corner, high_corner, num_cells, halo_width );
+    auto particles =
+        std::make_shared<CabanaPD::Particles<memory_space, model_type,
+                                             typename thermal_type::base_type>>(
```

Comment:
What is this change for ("typename thermal_type::base_type")?

Reply:
This can be hidden from the user, but it comes down to heat transfer simulations needing the same temperature-dependent properties as static temperature simulations

Reply:
I see.